light-client/ibc-client-starknet/src/client_state/execution.rs

```diff
@@ -61,6 +61,7 @@ where
 
         let new_client_state = ClientStateType {
             latest_height: header.height,
+            chain_id: "dummy".parse()?, // TODO: fill with correct info
```

Comment:
I think we need to query the current client state here, and then use the chain ID from the original client state.

Reply:
Done in 279deb4858789e3726fa5eaf7af36d2cb65af870